From: npiggin@suse.de
Subject: disable smtnice
References: 163880
Patch-mainline: obsolete

SLES9 has SMT nice disabled for different reasons, but in 
sles10, it creates significant contention on the runq
locks.  Any scheduler intensive benchmark is slow with
smt nice enabled.  This patch turns it off.

Acked-by: mason@suse.de
Acked-by: okir@suse.de

diff -r f0bffad73506 include/linux/topology.h
--- a/include/linux/topology.h	Thu May 18 15:55:43 2006 -0400
+++ b/include/linux/topology.h	Wed May 24 10:05:06 2006 -0400
@@ -105,8 +105,7 @@
 				| SD_BALANCE_NEWIDLE	\
 				| SD_BALANCE_EXEC	\
 				| SD_WAKE_AFFINE	\
-				| SD_WAKE_IDLE		\
-				| SD_SHARE_CPUPOWER,	\
+				| SD_WAKE_IDLE,		\
 	.last_balance		= jiffies,		\
 	.balance_interval	= 1,			\
 	.nr_balance_failed	= 0,			\

